lib/internal/util.js

```diff
@@ -4,7 +4,7 @@ const errors = require('internal/errors');
 const binding = process.binding('util');
 const signals = process.binding('constants').os.signals;
 
-const { createPromise, promiseResolve, promiseReject } = binding;
+const { createPromise, promiseReject, promiseResolve } = binding;
```

Comment:
These changes seem to be unrelated...not that it affects this PR tho

Reply:
@joyeecheung Ah, yes, leftover organizing/alphabetizing from a previous version of this. I'll remove these changes. Thanks!